SourceryRuntime/Sources/Description.generated.swift

```diff
@@ -122,26 +122,17 @@ extension Method {
     override public var description: String {
         var string = "\(type(of: self)): "
         string += "name = \(String(describing: self.name)), "
-        string += "shortName = \(String(describing: self.shortName)), "
-        string += "callName = \(String(describing: self.callName)), "
+        string += "selectorName = \(String(describing: self.selectorName)), "
```

Comment:
Why we removed some strings from the descirption?

Reply:
@krzysztofzablocki should we maybe improve it with your https://github.com/krzysztofzablocki/Difference ?

Reply:
Is that a performance downgrade or what specifically? I thought it was useful to show them when you print an object.

Reply:
It's more of an issue in tests. Not sure how much printing descriptions in templates is used, probably only during development. I would say that docs should be good enough to not need to inspect properties like that, but I can see your point. @krzysztofzablocki what do you think? Maybe if we use Difference we can leave computed properties in description and still have more readable failure descriptions in tests?

Reply:
I'd keep it as it is, with removed computed properties. Equality wouldn't even compare them so no point in diffing them, good call.

Our real implementation is better than Difference as it uses real AST and specialized implementation. Difference is for those people that don't use Sourcery for some bizarre and completely wrong reason 😹